### PR TITLE
[Refactor] Rename RssConf to CelebornConf.

### LINKED ISSUE #801: flusher + file writer

### PR DESCRIPTION
**Pull request by FMX**
# [BUG]/[FEATURE] title

### How was this patch tested?

/cc @related-reviewer

/assign @main-reviewer

### CODE REVIEW

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala [hunk 1]
```diff
@@ -1116,15 +1116,27 @@ object RssConf extends Logging {
...
+      .createWithDefaultString("120s")
 
-  def fileWriterTimeoutMs(conf: RssConf): Long = {
-    conf.getTimeAsMs("rss.filewriter.timeout", "120s")
-  }
+  val WORKER_FILEWRITE_CLOSE_TIMEOUT: ConfigEntry[Long] =
+    buildConf("celeborn.worker.filewriter.close.timeout")
+      .withAlternative("rss.filewriter.timeout")
+      .categories("worker")
+      .doc("Timeout for a file writer to close")
```
Comment (AngersZhuuuu) [inline common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1133]: .version("0.2.0")

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala [hunk 2]
```diff
@@ -1180,17 +1192,45 @@ object RssConf extends Logging {
...
+
+  val WORKER_DISK_RESERVATION: ConfigEntry[Long] =
+    buildConf("celeborn.worker.disk.reservation")
+      .withAlternative("rss.disk.minimum.reserve.size")
+      .categories("worker")
+      .doc("Celeborn worker reserved space for each disk.")
+      .bytesConf(ByteUnit.BYTE)
+      .createWithDefaultString("5G")
+
+  def HDDFlusherThread(conf: RssConf): Int = conf.get(WORKER_FLUSHER_HDD_THREAD_COUNT)
```
Comment (AngersZhuuuu) [inline common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1227]: We can move such method to `RssConf` class then just call `conf. HDDFlusherThread()`

**Comment by AngersZhuuuu** [comment]
Run `UPDATE=1 build/mvn test -pl common -am -Pspark-3.3 -Dtest=none -DwildcardSuites=org.apache.celeborn.ConfigurationSuite`

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala (hunk 1 above)
Comment (FMX) [reply common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1132]: Ok.

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala (hunk 2 above)
Comment (FMX) [reply common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1250]: Sounds good to me.

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala [hunk 3]
```diff
@@ -1180,17 +1191,77 @@ object RssConf extends Logging {
     }
   }
 
-  def HDDFlusherThread(conf: RssConf): Int = {
-    conf.getInt("rss.flusher.hdd.thread.count", 1)
-  }
+  val WORKER_FLUSHER_HDD_THREAD_COUNT: ConfigEntry[Int] =
+    buildConf("celeborn.worker.flusher.hdd.thread.count")
```
Comment (pan3793) [inline common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1195]: As previously discussed, we should unify "thread.count", "numThreads" and so on to "threads"

Comment (FMX) [reply common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1195]: OK， I'll change it.

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala [hunk 4]
```diff
@@ -362,6 +363,102 @@ class RssConf(loadDefaults: Boolean) extends Cloneable with Logging with Seriali
...
+      val prefix = workerBaseDirPrefix
+      val number = workerBaseDirNumber
+      (1 to number).map { i =>
+        (s"$prefix$i", defaultMaxCapacity, HDDFlusherThread(), HDD)
+      }
+    }
+  }
+
+  def partitionSplitMinimumSize(): Long = {
+    getSizeAsBytes("rss.partition.split.minimum.size", "1m")
```
Comment (AngersZhuuuu) [inline common/src/main/scala/org/apache/celeborn/common/RssConf.scala:448]: Seems not changed?

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala [hunk 5]
```diff
@@ -1102,29 +1199,38 @@ object RssConf extends Logging {
...
-  val WORKER_FLUSH_BUFFER_SIZE: ConfigEntry[Long] =
-    buildConf("celeborn.worker.flush.buffer.size")
+  val WORKER_FLUSHER_BUFFER_SIZE: ConfigEntry[Long] =
+    buildConf("celeborn.worker.flusher.buffer.size")
       .withAlternative("rss.worker.flush.buffer.size")
       .categories("worker")
       .doc("Size of buffer used by a single flusher.")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("256k")
-
```
Comment (AngersZhuuuu) [inline common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1112]: blank line?

File: common/src/main/scala/org/apache/celeborn/common/RssConf.scala [hunk 6]
```diff
@@ -1102,29 +1199,38 @@ object RssConf extends Logging {
...
   def workerHeartbeatTimeoutMs(conf: RssConf): Long = conf.get(WORKER_HEARTBEAT_TIMEOUT)
 
   def workerReplicateThreads(conf: RssConf): Int = conf.get(WORKER_REPLICATE_THREADS)
 
   def workerCommitThreads(conf: RssConf): Int = conf.get(WORKER_COMMIT_THREADS)
 
-  def workerFlushBufferSize(conf: RssConf): Long = conf.get(WORKER_FLUSH_BUFFER_SIZE)
+  def workerFlusherBufferSize(conf: RssConf): Long = conf.get(WORKER_FLUSHER_BUFFER_SIZE)
```
Comment (AngersZhuuuu) [inline common/src/main/scala/org/apache/celeborn/common/RssConf.scala:1215]: can move to `RssConf` class

File: worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala [hunk 7]
```diff
@@ -321,7 +320,7 @@ object DeviceMonitor {
       val freeSpace = usage(usage.length - 3)
       val used_percent = usage(usage.length - 2)
 
-      val status = freeSpace.toLong < RssConf.diskMinimumReserveSize(rssConf) / 1024 / 1024 / 1024
+      val status = freeSpace.toLong < rssConf.diskMinimumReserveSize / 1024 / 1024 / 1024
```
Comment (AngersZhuuuu) [inline worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala:323]: I think we should rename rssConf to conf

Comment (FMX) [reply worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala:323]: OK. I'll change it once and for all. 😎

Comment (AngersZhuuuu) [reply worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala:323]: > OK. I'll change it once and for all. 😎

Also rename RssConf to CelebornConf?

Comment (FMX) [reply worker/src/main/scala/org/apache/celeborn/service/deploy/worker/storage/DeviceMonitor.scala]: Sounds great. I'll change it.

**Comment by FMX** [comment]
This pr will hold until #813 is done.

**Pull request closed without merging**